shared/more/style-sheet.render.native.js

```diff
@@ -282,10 +316,19 @@ export default class Render extends Component {
     this.setState({check: next})
   }
 
+  _selectTab (idx: number) {
+    this.setState({tabSelected: [idx === 0, idx === 1, idx === 2]})
```

Comment:
no, this is the dumb component, just showing what you can do. just add more TabBar.Items 
in practice this doens't horizontally scroll or anything so we'll have to see how it'll have to evolve. right now i think we only ever have 2 tabs. public/private folders and switching input modes for provisioning. it takes a tabWidth but maybe  it should take all the width? we can change it later